test/Microsoft.Azure.WebJobs.Host.FunctionalTests/PublicSurfaceTests.cs

```diff
@@ -21,12 +21,10 @@ public void WebJobs_Host_Storage_VerifyPublicSurfaceArea()
 
             var expected = new[]
             {
-                "CloudBlobContainerDistributedLockManager",
-                "DistributedLockManagerContainerProvider",
                 "JobHostInternalStorageOptions",
                 "RuntimeStorageWebJobsBuilderExtensions",
-                "StorageBaseDistributedLockManager",
-                "StorageServiceCollectionExtensions"
+                "StorageServiceCollectionExtensions",
```

Comment:
@fabiocav, @mathewc need sign off on IAzureStorageProvider public type.

Reply:
Yes, we need to do a detailed review on all this new public surface area